pkg/BUILD

```diff
@@ -8,6 +8,21 @@ exports_files(
     visibility = ["//visibility:public"],
 )
 
+config_setting(
+    name = "have_rpmbuild",
+    flag_values = {
+        "//toolchains:is_rpmbuild_available": "1",
+    },
+    visibility = ["//visibility:private"],
+)
+
+constraint_setting(name = "not_compatible_setting")
```

Comment:
@gregestren , @philsc, Is it time to put this somewhere common?  Or is it fine duplicating this?  It is a pretty handy trick :)

Reply:
Or make it built-in? 

> Pretty. :) Comments are more or less nitpicky.
> 
> We should turn the RPM tests back on at some point; I got distracted and never did...

This is not quite ready for review. I just created it to show @austinschuh and @philsc a point about incompatible target skipping based on toolchains.

With this CL, bazel test ... on a mac does, correctly, skip the rpm tests. Except for the manifest test. That only depends on the rule, not rpmbuild, so it runs, but the bash script can not find md5sum.  #255

Reply:
Maybe @tools/config/common_constraints.bzl.
The reusable name will make all examples easier, reduce boilerplate and steer people away from accidental inventiveness.

Reply:
https://github.com/bazelbuild/platforms?

It certainly wouldn't hurt to have a standard reference.